src/main/java/com/google/dicomwebfuse/fuse/DicomPathParser.java

```diff
@@ -46,8 +46,14 @@ DicomPath parsePath(String path, Command command) throws DicomFuseException {
     DicomPath tempDicomPath;
     switch (pathRESTLength) {
       case 1:
+        String dicomStoreName;
+        if(pathREST[0].contains(" ")) {
+          dicomStoreName = pathREST[0].replace(" ","_");
```

Comment:
I'm assuming this is to support directory names with a " " in them, since the Healthcare API doesn't allow spaces in them?

Do we need this? Can we just return an error to the user instead of renaming?

Reply:
> I'm assuming this is to support directory names with a " " in them, since the Healthcare API doesn't allow spaces in them?

Healthcare API doesn't support white space in the DICOM Store name. Only "." or "_" or "-".

> Do we need this? Can we just return an error to the user instead of renaming?

Yes, we need this because macOS or Windows create a folder with white space by default in the graphical interface.

Reply:
Ok can you please add a comment here explaining this?

Reply:
Done